compiler/dcalc/interpreter.ml

```diff
@@ -212,48 +211,44 @@ let rec evaluate_operator (ctx : Ast.decl_ctx) (op : A.operator Pos.marked)
         if !Cli.trace_flag then (
           match entry with
           | VarDef _ ->
-              Cli.log_print
-                (Format.asprintf "%*s%a %a: %s" (!log_indent * 2) "" Print.format_log_entry entry
-                   Print.format_uid_list infos
-                   (match e' with
-                   (* | Ast.EAbs _ -> Cli.print_with_style [ ANSITerminal.green ] "<function>" *)
-                   | _ ->
-                       let expr_str =
-                         Format.asprintf "%a" (Print.format_expr ctx ~debug:false) (e', Pos.no_pos)
-                       in
-                       let expr_str =
-                         Re.Pcre.substitute ~rex:(Re.Pcre.regexp "\n\\s*")
-                           ~subst:(fun _ -> " ")
-                           expr_str
-                       in
-                       Cli.print_with_style [ ANSITerminal.green ] "%s" expr_str))
+              (* FIXME: this usage of Format is broken, Formatting requires that all is formatted in
```

Comment:
I've used `TODO` accross the codebase rather than `FIXME`, I guess we can stick with one keyword only :)